packages/database/test/helpers/util.ts

```diff
@@ -243,3 +267,16 @@ export function testRepoInfo(url) {
   const [, ns] = match;
   return new ConnectionTarget(`${ns}.firebaseio.com`, true, ns, false);
 }
+
+export function repoInfoForConnectionTest() {
+  if (USE_EMULATOR) {
+    return new ConnectionTarget(
+      `localhost:${EMULATOR_PORT}`,
+      false, // emulator does not support https or wss
```

Comment:
can you document what this parameter is named (probably something like `/* useSsl = */ false,`)

Reply:
Done.